plugins/shared-react/package.json

```diff
@@ -23,7 +23,9 @@
   },
   "dependencies": {
     "@kubernetes/client-node": "^0.18.1",
-    "classnames": "2.3.2"
+    "classnames": "2.3.2",
+    "date-fns": "^2.30.0",
+    "mathjs": "^11.9.1"
```

Comment:
shall we have `^` here or use fixed version?

Reply:
Only the dev dependencies need to be fixed. e.g. `3scale-backend` [`package.json`](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/3scale-backend/package.json)